### PR TITLE
DEVDOCS-2391: Added Contextual filters article

### PR DESCRIPTION
**Pull request by ektravel**
# [DEVDOCS-2391](https://jira.bigcommerce.com/browse/DEVDOCS-2391)

## What changed?
* added a new article
* added new images

### CODE REVIEW

File: docs/api-docs/management-apis/tutorials/contextual-filters-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+                    "show_in_stock_filter": true
+                }
+            ]
+        },
+    ],
+    "meta": {}
+}
+```
+
+The order and names of returned filters should match what is displayed under Category Filters in the control panel.
```
Comment (traci707) [inline docs/api-docs/management-apis/tutorials/contextual-filters-tutorial.md:191]: The order and names of returned filters should match the filters listed under Category Filters in the control panel.

File: docs/api-docs/management-apis/tutorials/contextual-filters-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+           "type": "price",
+           "collapsed_by_default": false,
+           "is_enabled": true
+       }
+       ...
+   ],
+   "meta": {}
+}
+```
+The global filters returned in the response of the `GET` request should match the filters you see in the control panel under **Storefront Filters**. This set of filters is what you currently have configured on your store.
```
Comment (traci707) [inline docs/api-docs/management-apis/tutorials/contextual-filters-tutorial.md:85]: The global filters returned from completing a `GET` request should match the filters you see in the control panel under **Storefront Filters**. This set of filters is what you currently have configured on your store.

**Review by traci707 — COMMENTED** [review]
I made a few suggestions. 
- I changed curly quotation marks to straight.
- I added on to the prerequisites to prevent duplication of text
- Removed passive voice
- Is there a way to test the examples in the request runner?

**Comment by ektravel** [comment]
Closing this PR to fix conflicts.

**Pull request closed without merging**